tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -2344,4 +2344,212 @@ describe("Service Form Modal", () => {
       });
     });
   });
+  describe("Vertical Bursting", () => {
+    beforeEach(() => {
+      cy.configureCluster({
+        mesos: "1-empty-group",
+        nodeHealth: true
+      });
+      cy.visitUrl({ url: "/services/overview/create" });
+    });
+    describe("Single Container", () => {
+      it("is available in the Form", () => {
+        cy.contains("Single Container").click();
+        cy.contains("More Settings").click();
+        cy.root()
+          .getFormGroupInputFor("CPUs")
+          .filter("input[name='limits.cpus']")
+          .type("{selectall}1");
```

Comment:
why do we need to empty the form input?

Reply:
I want to make sure that I definitely only set 1. By this I am making sure to remove any value if there is even one. This is covering the case if we ever consider adding a default value for this.